typo3Model.xml

```diff
@@ -41,6 +41,27 @@
       <type name="dkd:typo3:pages">
           <title>TYPO3 Page</title>
           <parent>cm:folder</parent>
+          <properties>
+              <property name="dkd:typo3:pages:doktype">
+                  <type>d:int</type>
+                  <mandatory>true</mandatory>
+                  <multiple>false</multiple>
+              </property>
+          </properties>
+          <associations>
+              <association name="dkd:typo3:pages:shortcut_to">
```

Comment:
I think that is a bad idea. Pages should be connected with a relationship between them